celements-scheduler/.gitignore

```diff
@@ -0,0 +1,12 @@
+*.class
+
+# Package Files #
+*.jar
+*.war
+*.ear
+/target
+/.classpath
+/.project
+/.settings
+/xwiki.log
+/src/main/java/META-INF
```

Comment:
move .gitignore to repository root, see e.g. https://github.com/celements/celements-xwiki/blob/dev/.gitignore